### PR TITLE
Alessio reset invalid license

### LINKED ISSUE #225: Add an additional way to activate the user in the backend
#### Steps to reproduce the issue

If an error occurs during the plugin activation users cannot "activate" themselves within the backend, meaning that they will not be registered in the backend database, and cannot use any of the plugin functionality because they are missing a license key.

There should be an additional way to activate the user, even after the plugin activation.

### LINKED ISSUE #230: Error management of invalid licenses
#### What I expected

when the user enters an invalid license, the data must be reset and a new license must be reassigned

### PR DESCRIPTION
**Pull request by cpalessio96**
### All Submissions:

*   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
*   [x] Does your code has proper inline documentation? 

### Changes proposed in this Pull Request:
when the user enters an invalid license, the data must be reset and a new license must be reassigned

Closes #225  .
Closes #230 .

### CODE REVIEW

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- includes/Core/Cron.php  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5920686&bid=19400617)

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- includes/Core/Cron.php  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5920686&bid=19400617)

**Review by HyprCoder — DISMISSED** [review]

File: includes/Api/ApiLicenseValidation.php [hunk 1]
```diff
@@ -42,6 +43,69 @@ public function register_license_validation_route() {
...
+	 *
+	 * @param \WP_REST_Request $request valori della richiesta.
+	 * @return WP_REST_Response|array<mixed>
+	 */
+	public function get_init_data( $request ) {
+		$headers           = $request->get_headers();
+		$token             = $headers['token'][0];
+		$current_token     = get_option( 'wubtitle_token' );
+		$token_time        = get_option( 'wubtitle_token_time' );
+		$token_exipiration = $token_time + ( 60 * 5 );
```
Comment (HyprCoder) [inline includes/Api/ApiLicenseValidation.php]: The expiration should be set when you're are saving it in the database

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- includes/Core/Cron.php  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5920686&bid=19400617)

**Review by HyprCoder — CHANGES_REQUESTED** [review]

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- includes/Core/Cron.php  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5920686&bid=19400617)

**Review by HyprCoder — APPROVED** [review]

**Comment by GiuseppeIuculano** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- includes/Core/Cron.php  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5920686&bid=19400617)

**Pull request merged**